### ISSUE TITLE
No specific subclasses for fields?

### ISSUE THREAD

**Issue by daybyter** [penberg/falcon#12]
Hi!

There are no Field subclasses for Field for specific fieldsm like NoRelatedSym, MDReqID etc? 
Is this intentionally, so you have to use new Field( <tag>, <value>) ?
Make the sources no very usable, I think? I wrote me a simple code generator in Python to create subclasses from a simple text file.
Maybe this costs _some_ performance, but creates nicer sources?

Ciao,
Andreas

**Comment by penberg** [comment]
@daybyter I strongly recommend you have a look at [Philadelphia FIX engine](https://github.com/paritytrading/philadelphia), which is much more feature complete and probably faster too.

/cc @jvirtanen

**Comment by jvirtanen** [comment]
@daybyter Yes, you might want to take a look at [Philadelphia](https://github.com/paritytrading/philadelphia/tree/improvements/project-structure), which is designed based on Falcon and [Libtrading](https://github.com/libtrading/libtrading). It is currently more complete than Falcon, and contains definitions for all FIX 4.2, FIX 4.3, FIX 4.4, FIX 5.0, FIX 5.0 SP1, and FIX 5.0 SP2 message types, tags, and enumerations.

I have prepared [a small example](https://github.com/jvirtanen/gdax-fix-example) that shows how to log onto [GDAX](https://www.gdax.com) using Philadelphia.

**Comment by daybyter** [comment]
Thanks a lot for your hints! I'll take a look.
